binding/bucket.go

```diff
@@ -34,3 +58,57 @@ func (h *Bucket) Delete(path string) (err error) {
 	err = h.Client.Delete(pathlib.Join(h.path, path))
 	return
 }
+
+func (h *Bucket) GetDir(file io.Reader, path string) (err error) {
+	err = h.Client.getDir(file, path)
+	return
+}
+
+func (h *Bucket) PutDir(file io.Writer, path string) (err error) {
+	err = h.Client.putDir(file, path)
+	return
+}
+
+func (h *Bucket) Compress(src string, buf io.Writer) error {
```

Comment:
I'm struggling to understand how we are testing the API with this method.

Reply:
This is just the compress method. Shall I add it to the `api/bucket,go`?